migrations/Version202307041134591871_taoTestCenter.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2023 (original work) Open Assessment Technologies SA;
+ *
+ *
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoTestCenter\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\Exception\IrreversibleMigration;
+use oat\generis\persistence\PersistenceManager;
+use oat\oatbox\reporting\Report as Report;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+
+/**
+ * phpcs:disable Squiz.Classes.ValidClassName
+ */
+final class Version202307041134591871_taoTestCenter extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Fixes field name for ByPassProctor to a proper one (typo fix).';
+    }
+
+    public function up(Schema $schema): void
+    {
+        try {
+            $persistence = $this->getPersistence();
+
+            $query = sprintf(
```

Comment:
No other migration ever executes queries directly on `statements` table. I don't have a better suggestion without making it complicated, but lets check that SQL is used for ontology

Reply:
I know it looks unusual, and I've asked folks if anybody could suggest a better approach, however, nobody faced something like that and the only suggestion was to leave it as is if it not hurts. But unfortunately, it is brakes the import, so I need to change it. 
As for the query itself, it is rather simple and ANSI SQL compatible so should work on any compatible engine (MySQL, PgSQL, MSSQL, etc.). Could you clarify which ontology SQL we should check?

Reply:
Correct, this change is preparation for migration, so it has to run before neo4j migration. Unfortunately, I'm not sure what is the best way to ensure it, as fix and migration scripts in 2 different extensions.